test/integration/examples/tls_t/tls_t.go

```diff
@@ -742,12 +785,14 @@ func SendReceive(addr string, options []string, seek string) error {
 				// We're done here, so we just return whatever err we got before
 				return err
 			}
-			log.Printf("  %v", line)
+			log.Printf("%v:  %v", name, line)
 		case <-timeoutCh:
 			if err != nil {
-				log.Printf("timed out waiting for SendReceive, but received an error befor: %v", err)
+				return fmt.Errorf("%v: timed out on post-test stdout flush, but received an error before: %w", name, err)
 			}
-			return fmt.Errorf("timed out waiting for SendReceive function to finish")
+			// We do not consider this as an error, as it is post-test, but worthy to note
+			log.Printf("%v: timed out on post-test stdout flush", name)
```

Comment:
This could be anything; it could be Skupper, or openssl's `s_client`/`s_server`, or the network.  This is more of a safety net; we do not expect to fall on it, but we can't let the test just keep running forever, in case such a timeout happens, as we're depending on an external binary.  If it does, we detected it ourselves, so there is no error to report.  All we have for debugging is the OpenSSL messages that have already been printed before.